report/main.qmd

```diff
@@ -0,0 +1,139 @@
+
+However, ANOVA's limitation lies in its inability to account for external variables that could influence outcomes, such as baseline consumption patterns or hospital site characteristics. Despite this, ANOVA remains valuable for initial analysis, with its limitations highlighting the need for a comprehensive approach, possibly incorporating Analysis of Covariance (ANCOVA), to fully understand the interventions’ impacts on beverage selection.
+
+## Linear Mixed-Effect Model
+
+Linear Mixed-Effects (LME) model is a statistical model that accounts for fixed effects, common trends that are present at all levels of the data, as well as random effects, correlation within groups and variation between groups. The inclusion of random effects makes LME especially adept at handling hierarchical and longitudinal data, where observations are grouped into different levels. In these models, we assume that the errors (or the differences between the predicted value and the actual value) are random and follow a normal distribution with an average of zero. 
+
+## Recommendation
+
+For the first and second questions, we recommend using the following ANOVA model 
+$$\text{Sales} = \beta_0+\beta_1\times\text{Intervention}+\beta_2\times\text{Site}$$ 
+with the pre-intervention period being the reference. For the third question, we recommend using the following ANOVA model 
+$$\text{Sales} = \beta_0+\beta_1\times\text{Calorie } +\beta_2\times\text{Exercise}$$
+with the combination period being the reference. If the interest is also the effect of combining discount and messaging, then the following model is recommended 
+$$\text{Sales} = \beta_0+\beta_1\times\text{Discount }$$ 
+with the discount plus messaging as the reference. For the final question, we recommend using the following ANOVA model 
+$$\text{Sales} = \beta_0+\beta_1\times\text{Calorie}$$
+with the calorie admonishment period being the reference. 
+
+In the model for the second question, we recommend first running a preliminary ANOVA for each of the interventions and the pre-intervention period. If there are significant differences in estimated coefficients between interventions, a LME model with intervention-dependent random effects is recommended. A similar strategy needs to be applied to the other questions. Still, the preliminary ANOVA is performed for each site and site-dependent random effects are added when differences are significant. The reason for these preliminary analyses is to determine if LME is necessary or not. Note that the sales in these models refer to the trend extracted in the Pre-analysis step. After the models are fitted, the residuals need to be visualized via scatter plot, autocorrelation plot and QQ-plot to confirm independent, equal variance and normal error assumptions for both ANOVA and LME.
```

Comment:
I would suggest you to insert this "recommendation" section into your analysis process. If you place it between formal analysis and conclusion, it seems to be slightly redundant.